src/main/java/com/ericsson/ei/rmqhandler/RmqHandler.java

```diff
@@ -117,6 +112,7 @@
     private SimpleMessageListenerContainer container;
     private SimpleMessageListenerContainer waitlistContainer;
 
+
     @Bean
     ConnectionFactory connectionFactory() {
         com.rabbitmq.client.ConnectionFactory connectionFactory = new com.rabbitmq.client.ConnectionFactory();
```

Comment:
Please remove the package from the type. Nicer to have it in imports. I do not know how we managed to have it here.

Reply:
It seems this is a workaround for using org.springframework.amqp.rabbit.connection.CachingConnectionFactory and com.rabbitmq.client.ConnectionFactory together as they are otherwise incompatible.

Reply:
no it is not.

This star import 

import org.springframework.amqp.rabbit.connection.*;

needs to be replaced with:

import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
import com.rabbitmq.client.ConnectionFactory;

Reply:
The implementation as it is now requires both com.rabbitmq.client.ConnectionFactory  and org.springframework.amqp.rabbit.connection.ConnectionFactory. It is not enough to change the imports as different connectionfactories are used in the code. An option is to rewrite the whole implementation but that should maybe me moved to another PR. I've changed the naming of the different packages used to hopefully clarify which is what in the code.